spec/hardware/memory.fmf

```diff
@@ -0,0 +1,8 @@
+summary:
+    Select the desired amount of memory
+example:
+  - |
+    # Basic key-value format used to specify the memory size
+    memory: 8 GB
```

Comment:
I'd also add an example to demonstrate "at least" requirement, `>=`. It might be impractical to request the exact amount of RAM, most of the time we're fine with getting at least the required anyway.

Reply:
Good idea, fixed in 1cf0d3b2.